src/index.html

```diff
@@ -10,13 +10,17 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Stop watch</title>
+    <title>Stop stopwatch</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element name. Instead of 'stopwatch--speed-up', it should be 'stopwatch stopwatch--speed-up' if 'speed-up' is a modifier of 'stopwatch'.